README.rst

```diff
@@ -69,6 +69,168 @@ The *apidoc* extension uses the following configuration values:
 
    **Optional**, defaults to ``False``.
 
+Migration from pbr
+------------------
+
+`pbr`_ has historically included a `custom variant <build_sphinx>`_ of the
+``build_sphinx`` distutils command. This provides, among other things, the
+ability to generate API documentation as part of build process. Clearly this is
+not necessary with this extension.
+
+There are two implementations of the API documentation feature in *pbr*:
+*autodoc_tree* and *autodoc*. To describe the difference, let's explore how one
```

Comment:
Maybe it would be a good idea at this point to mention how you can tell which implementation you might be using (i.e. looking for `autodoc_tree_index_modules = True` vs. `autodoc_index_modules = True` in setup.cfg)

Reply:
Done